lustre_exporter_test.go

```diff
@@ -114,6 +114,55 @@ func toggleCollectors(target string) {
 	}
 }
 
+func stringAlphabetize(str1 string, str2 string) (int, error) {
+	letterCount := 0
+	if len(str1) > len(str2) {
+		letterCount = len(str2)
+	} else {
+		letterCount = len(str1)
+	}
+
+	for i := 0; i < letterCount-1; i++ {
+		if str1[i] == str2[i] {
+			continue
+		} else if str1[i] > str2[i] {
+			return 1, nil
+		} else {
+			return 2, nil
+		}
+	}
+
+	return 0, errors.New("Strings are too similar")
```

Comment:
Regardless of if I could write the code to make it pass, we're in a bad situation if we have duplicate labels...so I just decided to return an error. It should fail, yes. Perhaps I should write unit tests...for unit tests... >_>